google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -208,17 +244,35 @@ public static EnhancedBigtableStub create(EnhancedBigtableStubSettings settings)
    *       implementation can be configured in by the {@code rowAdapter} parameter.
    *   <li>Retry/resume on failure.
    *   <li>Filter out marker rows.
+   *   <li>Add tracing & metrics.
    * </ul>
    */
   public <RowT> UnaryCallable<Query, RowT> createReadRowCallable(RowAdapter<RowT> rowAdapter) {
-    return createReadRowsCallable(
-            ServerStreamingCallSettings.<Query, Row>newBuilder()
-                .setRetryableCodes(settings.readRowSettings().getRetryableCodes())
-                .setRetrySettings(settings.readRowSettings().getRetrySettings())
-                .setIdleTimeout(settings.readRowSettings().getRetrySettings().getTotalTimeout())
-                .build(),
-            rowAdapter)
-        .first();
+    UnaryCallable<Query, RowT> readRowCallable =
+        createReadRowsBaseCallable(
+                ServerStreamingCallSettings.<Query, Row>newBuilder()
+                    .setRetryableCodes(settings.readRowSettings().getRetryableCodes())
+                    .setRetrySettings(settings.readRowSettings().getRetrySettings())
+                    .setIdleTimeout(settings.readRowSettings().getRetrySettings().getTotalTimeout())
+                    .build(),
+                rowAdapter)
+            .first();
+
+    UnaryCallable<Query, RowT> traced =
+        new TracedUnaryCallable<>(
+            readRowCallable,
+            clientContext.getTracerFactory(),
+            SpanName.of(TRACING_OUTER_CLIENT_NAME, "ReadRows"));
+
+    UnaryCallable<Query, RowT> measured =
+        new MeasuredUnaryCallable<>(
+            traced,
+            TRACING_OUTER_CLIENT_NAME + ".ReadRows",
+            tagger,
+            statsRecorder,
+            clientContext.getClock());
+
+    return measured.withDefaultCallContext(clientContext.getDefaultCallContext());
```

Comment:
Can we use `return createUserFacingUnaryCallable("ReadRow", readRowCallable);` ?